web/libs/datamanager/src/sdk/lsf-sdk.js

```diff
@@ -155,6 +155,10 @@ export class LSFWrapper {
       interfaces.push("comments:resolve-any");
     }
 
+    if (this.project.review_settings.require_comment_on_reject) {
```

Comment:
i see this flag being used in another spot that is also checking for `ff_front_dev_2186_comments_for_update` do we need to encapsulate this in this FF too?

Reply:
Great catch! For a second, I thought I needed to add it here, but then I saw how it had been implemented.

https://github.com/HumanSignal/label-studio/blob/develop/web/libs/datamanager/src/sdk/lsf-sdk.js#L124-L126
`FF_DEV_2186` is a string 🥲

Reply:
ahh, true thats not even checking it properly.
I also looked at who that FF is enabled for and it only raised more questions. seems to be on for a handful of users and off for every one else. need more info on what happened there. but as it stands - we should fix how this is being checked (probably another ticket)